src/navigation/ChooseWalletGroup.tsx

```diff
@@ -0,0 +1,225 @@
+              await backupUserDataIntoCloud({ wallets: newWallets });
+            } catch (error) {
+              logger.error(new RainbowError('[AddWalletSheet]: Updating wallet userdata failed after new account creation'), { error });
+              throw error;
+            }
+          }
+        } catch (e) {
+          logger.error(new RainbowError('[AddWalletSheet]: Error while trying to add account'), {
+            error: e,
+          });
+          showWalletErrorAlert();
+        }
+      },
+      profile: { color: null, name: `` },
+      type: 'wallet_profile',
+    });
+  };
+
+  return (
+    <ButtonPressAnimation onPress={onAddToGroup} scaleTo={0.95} style={{ flexDirection: 'row', alignItems: 'center', gap: 10 }}>
+      <View
+        style={[
+          { height: 40, width: 40, gap: 2, padding: 5 },
+          { borderRadius: 10, backgroundColor: separatorSecondary },
+          { flexWrap: 'wrap', flexDirection: 'row' },
+        ]}
+      >
+        {accounts.map(account => (
+          <AccountAvatar key={account.address} account={account} size={14} />
+        ))}
+      </View>
+      <View style={{ gap: 8 }}>
+        <Text color="label" size="15pt" weight="semibold">
+          {removeFirstEmojiFromString(wallet.name)}
+        </Text>
+        <View style={{ flexDirection: 'row', alignItems: 'center', gap: 5 }}>
+          <Text color="labelQuaternary" size="13pt" weight="bold">
+            {accounts[0].ens || formatAddressForDisplay(accounts[0].address, 4, 4)}
+          </Text>
+          {accounts.length > 1 && (
+            <View
+              style={[
+                { height: 16, paddingHorizontal: 3.5, justifyContent: 'center', marginVertical: -2 },
+                { borderColor: chroma(separatorSecondary).alpha(0.04).hex(), borderWidth: 1, borderRadius: 6 },
+                { backgroundColor: separatorSecondary },
+              ]}
+            >
+              <Text color="labelQuaternary" size="11pt" weight="bold">
+                {`+${accounts.length - 1}`}
+              </Text>
+            </View>
+          )}
+        </View>
+      </View>
+    </ButtonPressAnimation>
+  );
+}
+
+export function ChooseWalletGroup() {
+  const { wallets } = useWallets();
+
+  if (!wallets) return;
+
+  const groups = Object.values(wallets).filter(wallet => wallet.type === WalletTypes.mnemonic);
+
+  return (
+    <Box
+      height="full"
+      width="full"
+      background="surfaceSecondary"
+      style={{ paddingHorizontal: 24, paddingTop: 72, gap: 20, alignItems: 'center' }}
+    >
+      <Text color="label" size="22pt" weight="heavy">
+        {i18n.t(i18n.l.wallet.new.choose_wallet_group.title)}
+      </Text>
+      <Text color="labelQuaternary" size="15pt" weight="semibold" align="center">
+        {i18n.t(i18n.l.wallet.new.choose_wallet_group.description)}
+      </Text>
+      <Box
+        background="surfaceSecondaryElevated"
+        // shadow="12px" // TODO: adding shadow clips height (?)
```

Comment:
still needs to be done? If so, can help over slack if you need assistance with this